README.md

```diff
@@ -127,12 +126,12 @@ We get the following table of comparison.
 |degree|[10] => 5465008*q[1]^20 +…|5.453744 s(3.425 GiB)|0.000033 s ( 1.609 KiB)|
 
 
-simillary for The univariable polynomial, we have 
+similarly for The univariable polynomial, we have
```

Comment:
this sentence still is wrong, but I couldn't figure it out what you even meant, so I can't suggest an improvement (well I can guess that "univariable" is supposed to mean "univariate" and "The" -> "the" but the entire sentence still doesn't make sense to me. No need to explain it to me either!! But perhaps think about it and whether you can rephrase it.